docs/bit-scope.md

```diff
@@ -46,15 +46,15 @@ As shows in the [configurating bit](configurating-bit.md) section, you can list
 
 Just as you can use other Bit components as dependencies, we also understand that there's a lot of previous work that have been developed and deployed to other package managers, and code components should be able to depend on them. Bit does it by integrating to other known package source (for example - NPM). However, bit does not handle these dependencies, and will not manage them. If a code component has a package dependency defined, it will tell the user what needs to be installed. So unlike code component dependencies, you will be required to manage your external dependencies by yourself. Bit will help you with letting you know what's missing, but will not interfere with your dependency resolution process. 
 
-## Scope CI
+## Isolated Build/Test
 
-Bit comes with a built-in slim CI mechanism.
+Bit is able to run your components buid and test steps in an isolated environment.
```

Comment:
> Bit is able to run your components buid and test steps in an isolated environment.

Bit runs your **component's** **build** and test **operations** in an isolated environment.